test/protocol/http1/connection.rb

```diff
@@ -200,12 +200,15 @@
 	with '#read_response_body' do
 		with "GET" do
 			it "should ignore body for informational responses" do
-				expect(client.read_response_body("GET", 100, {'content-length' => '10'})).to be_nil
+				body = client.read_response_body("GET", 100, {'content-length' => '10'})
+				expect(body).to be_a(::Protocol::HTTP1::Body::Remainder)
```

Comment:
If we used this body in any way, wouldn't it corrupt the underlying connection?

Reply:
Yes and no. While not a dominant use case, one could read from the `Remainder` and it would simply end up returning the next set(s) of response headers followed by the body.

Think of `Remainder` in this context as a promise against the rest of the stream.